core/src/main/scala/quasar/plugin/postgres/datasource/Config.scala

```diff
@@ -21,19 +21,33 @@ import slamdata.Predef._
 import argonaut._, Argonaut._
 
 import java.net.URI
+import java.util.Properties
 
 import quasar.api.datasource.{DatasourceError, DatasourceType}
 import quasar.api.datasource.DatasourceError.InvalidConfiguration
 import quasar.plugin.postgres.datasource.PostgresCodecs._
 
+import org.postgresql.Driver
+
 import scalaz.NonEmptyList
 
 final case class Config(connectionUri: URI, connectionPoolSize: Option[Int]) {
+  import Config._
+
   @SuppressWarnings(Array("org.wartremover.warts.Null"))
   def sanitized: Config = {
     copy(connectionUri = Sanitization.sanitizeURI(connectionUri))
   }
 
+  def properties: Properties = {
+    val ps = Driver.parseURL(s"jdbc:${connectionUri.toString}", null)
```

Comment:
Should we check if this returned null? It's documented as `nullable` [here](https://jdbc.postgresql.org/documentation/publicapi/org/postgresql/Driver.html#parseURL-java.lang.String-java.util.Properties-).

Reply:
Good point! I've added `Option.apply`